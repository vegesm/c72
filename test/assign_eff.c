main() {
    extern iarr;
    int iarr[];
    int a,b;
    char c, d, carr[4];

    a = 4;
    printf("test1: %d\n", a);

    c = 15;
    printf("test2: %d\n", c);

    c = -128;
    printf("test3: %d\n", c);

    iarr[2] = 12345;
    printf("test4: %d %d %d\n", iarr[1], iarr[2], iarr[3]);

    carr[0] = 3;
    printf("test5: %d\n", carr[0]);
}

iarr[] 1, 8, 10, 128;

