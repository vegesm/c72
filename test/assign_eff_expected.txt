test1: 4
test2: 15
test3: -128
test4: 8 12345 128
test5: 3
